test/TestInternalGrains/PersistenceTestGrains.cs

```diff
@@ -668,9 +668,9 @@ private void CheckRuntimeEnvironment()
             var callStack = new StackTrace();
 
             executing = true;
-            Assert.AreEqual(_scheduler, scheduler, "Wrong TaskScheduler {0} Caller:{1}", scheduler, callStack);
-            Assert.IsNotNull(context, "Null ActivationContext -- Expected: {0} Caller:{1}", _context, callStack);
```

Comment:
to be fair, these were there as a precaution probably when developing, it's not like we expect to see them throw, and even if they do, xunit assertions will show a much more meaningfull exception message. The one that we should remove is the one that tests that is not null (or shift it down), as it's the only one that can "hide" information. I'll do that